strongbox-web-core/src/test/resources/etc/conf/strongbox-authorization.xml

```diff
@@ -1,36 +1,28 @@
 <?xml version="1.0" standalone="yes"?>
 <authorization-configuration>
-    <roles>
-        <!-- special treatment role -->
-        <role>
-            <name>ANONYMOUS_ROLE</name>
-            <description>Common anonymous user role</description>
-            <privileges>
-                <privilege>ARTIFACTS_RESOLVE</privilege>
-                <privilege>SEARCH_ARTIFACTS</privilege>
-            </privileges>
-        </role>
-        <role>
-            <name>USER_ROLE</name>
-            <description>Common user role</description>
-            <privileges>
-                <privilege>VIEW_USER</privilege>
-            </privileges>
-        </role>
-        <role>
-            <name>CUSTOM_ROLE</name>
-            <description>Deployment role</description>
-            <privileges>
-                <privilege>Deploy</privilege>
-            </privileges>
-        </role>
-    </roles>
-    <privileges>
-        <privilege>
-            <name>EXAMPLE_PRIVILEGE</name>
-            <description>This is an example of how you could add some custom privilege. Feel free to use it by name at
-                the REST API side to access restriction purposes.
-            </description>
-        </privilege>
-    </privileges>
+
+    <!-- special treatment role -->
+    <role>
+        <name>ANONYMOUS_ROLE</name>
+        <description>Common anonymous user role</description>
+        <privileges>
+            <privilege>ARTIFACTS_RESOLVE</privilege>
+            <privilege>SEARCH_ARTIFACTS</privilege>
+        </privileges>
+    </role>
+    <role>
+        <name>USER_ROLE</name>
+        <description>Common user role</description>
+        <privileges>
+            <privilege>VIEW_USER</privilege>
+        </privileges>
+    </role>
+    <role>
+        <name>CUSTOM_ROLE</name>
+        <description>Deployment role</description>
+        <privileges>
+            <privilege>Deploy</privilege>
+        </privileges>
+    </role>
+
```

Comment:
@ptirador 
we dont want to drop wrapper `<roles>` tag here, so the xml should still looks as it was like this:
```
    <roles>
        <!-- special treatment role -->
        <role>
            <name>ANONYMOUS_ROLE</name>
            <description>Common anonymous user role</description>
            <privileges>
                <privilege>ARTIFACTS_RESOLVE</privilege>
                <privilege>SEARCH_ARTIFACTS</privilege>
            </privileges>
        </role>
        <role>
            <name>USER_ROLE</name>
            <description>Common user role</description>
            <privileges>
                <privilege>VIEW_USER</privilege>
            </privileges>
        </role>
        <role>
            <name>CUSTOM_ROLE</name>
            <description>Deployment role</description>
            <privileges>
                <privilege>Deploy</privilege>
            </privileges>
        </role>
    </roles>
    <privileges>
        <privilege>
            <name>EXAMPLE_PRIVILEGE</name>
            <description>This is an example of how you could add some custom privilege. Feel free to use it by name at
                the REST API side to access restriction purposes.
            </description>
        </privilege>
    </privileges>
```

also seems that you dropped all the `<privelegies>` here too

Reply:
you can try to use ` @XmlElementWrapper(name = "roles")`  like this:
```
    @XmlElementWrapper(name = "roles")
    @XmlElement(name = "role")
    @Embedded
    private Set<Role> roles = new LinkedHashSet<>();
```

Reply:
@ptirador sorry for confusion about `<privelegies>`, seems that we really should remove it as there is no such field in `AuthorizationConfig`.